### ISSUE TITLE
ModuleNotFoundError: No module named 'enmapboxexampledata'

### ISSUE THREAD

**Issue by jakimowb** [EnMAP-Box/enmap-box#588]
Several unit tests fails due to this error, e.g. tests/enmap-box/enmapbox/coreapps/classfractionstatisticsapp

```
 _ ERROR collecting tests/enmap-box/enmapbox/coreapps/classfractionstatisticsapp/test_classfractionrenderer.py _
ImportError while importing test module '/usr/src/tests/enmap-box/enmapbox/coreapps/classfractionstatisticsapp/test_classfractionrenderer.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
../lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/enmap-box/enmapbox/coreapps/classfractionstatisticsapp/test_classfractionrenderer.py:10: in <module>
    from enmapboxtestdata import landcover_map_l3
../lib/python3/dist-packages/qgis/utils.py:888: in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
tests/src/enmapboxtestdata.py:7: in <module>
    import enmapboxexampledata.berlin
../lib/python3/dist-packages/qgis/utils.py:888: in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
E   ModuleNotFoundError: No module named 'enmapboxexampledata'
```

see https://github.com/EnMAP-Box/enmap-box/actions/runs/6037671279/job/16382393243 for more errors.

**Comment by janzandr** [comment]
_Discussed with @jakimowb._

The old Berlin dataset will be placed into the code repo under: 
enmap-box/tests/testdata/exampledata/berlin

In case the data will change again in the future, we will also move the current Potsdam data to:
enmap-box/tests/testdata/exampledata/potsdam

**Comment by janzandr** [comment]
Just to be clear.
1. In enmap-box/tests/src/**enmapboxtestdata**.py we do have localities inside the variable names
2. Only in enmap-box/enmapbox/**exampledata** we don't

No unittest should import from **enmapboxtestdata**, and thus, shouldn't be affected by this.

Anyways, I changed it in **exampledata**, as requested.

**Comment by jakimowb** [comment]
To my understanding unitests can use the standard exampledata as well, as this is often good enough to test the code

**Comment by janzandr** [comment]
> To my understanding unitests can use the standard exampledata as well

Yes, you can and should use the new Potsdam data. And you can and should also use the old Berlin data. Both can be imported via:
`from enmapboxtestdata import enmap_potsdam, enmap_berlin`

As requested, the new `enmap_berlin` also has the alias `enmap` now:
`from enmapboxtestdata import enmap`

What you did not need anymore inside a unittest, is to import from `enmapbox.exampledata`.

Hope that clarifies it.
